QSB/QSBNetworkManager.cs

```diff
@@ -113,12 +104,14 @@ public override void OnStartServer()
             if (WorldRegistry.OldDialogueTrees.Count == 0 && QSBSceneManager.IsInUniverse)
             {
                 WorldRegistry.OldDialogueTrees = Resources.FindObjectsOfTypeAll<CharacterDialogueTree>().ToList();
-                foreach (var item in Resources.FindObjectsOfTypeAll<FacePlayerWhenTalking>())
-                {
-                    item.gameObject.AddComponent<QSBFacePlayerWhenTalking>();
-                    Destroy(item);
-                }
             }
+
+            NetworkServer.UnregisterHandler(40);
+            NetworkServer.UnregisterHandler(41);
+            NetworkServer.UnregisterHandler(42);
+            NetworkServer.RegisterHandler(40, new NetworkMessageDelegate(QSBNetworkAnimator.OnAnimationServerMessage));
+            NetworkServer.RegisterHandler(41, new NetworkMessageDelegate(QSBNetworkAnimator.OnAnimationParametersServerMessage));
+            NetworkServer.RegisterHandler(42, new NetworkMessageDelegate(QSBNetworkAnimator.OnAnimationTriggerServerMessage));
```

Comment:
🤔
these magic numbers are in an enum somewhere, right?

Reply:
Yep - `eventType + MsgType.Highest`

Reply:
these aren't the highest numbers though. these are in built unity messages that i'm taking away from NetworkAnimator and adding to QSBNetworkAnimator